azure-iot-device/azure/iot/device/provisioning/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Azure Provisioning Device Library
+
+This library provides functionality that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring
+human intervention, enabling customers to provision millions of devices in a secure and scalable manner.
+
+"""
+from .sk_registration_client import RegistrationClient
+from .security import SymmetricKeySecurityClient
+from .models import RegistrationResult
+from .registration_client_factory import create_from_security_client
+
+__all__ = [
+    "RegistrationClient",
+    "SymmetricKeySecurityClient",
+    "RegistrationResult",
+    "create_from_security_client",
```

Comment:
>create_from_security_client" [](start = 5, length = 28)

I don't really want to be exposing bare functions like this, (which is why in IoTHub we've switched to importing auth, and then saying auth.from_connection_string), but again, we'll be getting rid of two-step instantiation soon, so I guess it's not a big deal for now.

Reply:
resolving the comment for now.

---
In reply to: [277780861](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r277780861) [](ancestors = 277780861)